### ISSUE TITLE
Does not show the keyboard shortcuts and React JSX is a bit weird

### ISSUE THREAD

**Issue by max-programming** [a5hk/night-coder#3]
I realized this when I got the main screen. The logo looks pretty dark and the keyboard shortcuts are not visible: 
![image](https://user-images.githubusercontent.com/51731966/83114676-e863d380-a0e6-11ea-8b84-0f008e2ad36d.png)

My overall experience was great. But the JSX looks a bit weird in simple tags: 
![image](https://user-images.githubusercontent.com/51731966/83114875-3082f600-a0e7-11ea-80c0-f5e97fc6b772.png)

I hope these suggestions help

**Comment by a5hk** [comment]
I am not sure if I can do anything about the logo. JSX should be good now:

![jsx](https://user-images.githubusercontent.com/5412540/83233406-e3188e80-a1a3-11ea-8655-230f146b2cb3.png)

**Comment by max-programming** [comment]
No I mean, the JSX is not looking like normal HTML file

**Comment by a5hk** [comment]
OK, I made some changes. First I used a color that has been used less frequently in JSX to differentiate HTML tags from other JSX code. Second I changed the color of opening and closing punctuation colors that has been used in HTML files.

![jsx](https://user-images.githubusercontent.com/5412540/83238470-b8cacf00-a1ab-11ea-86be-d70f67e3e04f.png)

I cannot make different html tag types to have different colors like normal HTML files, because they don't have different texmate scopes in JSX and TSX files. Let me know if there are other problems.
